proto/viam/service/sensors/v1/sensors.proto

```diff
@@ -10,49 +10,54 @@ option go_package = "go.viam.com/api/service/sensors/v1";
 option java_package = "com.viam.service.sensors.v1";
 
 // A SensorsService services keeps track of all sensors associated with a robot
+// This service is deprecated
 service SensorsService {
   // GetSensors returns the list of all sensors.
   rpc GetSensors(GetSensorsRequest) returns (GetSensorsResponse) {
     option (google.api.http) = {
       get: "/viam/api/v1/service/{name}/sensors"
     };
+    option deprecated = true;
   }
 
   // GetReadings returns the list of readings for all sensors specified.
   rpc GetReadings(GetReadingsRequest) returns (GetReadingsResponse) {
     option (google.api.http) = {
       get: "/viam/api/v1/service/sensors/{name}/readings"
     };
+    option deprecated = true;
   }
 
   // DoCommand sends/receives arbitrary commands
   rpc DoCommand(common.v1.DoCommandRequest) returns (common.v1.DoCommandResponse) {
     option (google.api.http) = {
       post: "/viam/api/v1/service/sensors/{name}/do_command"
     };
+    option deprecated = true;
   }
 }
 
+// The sensor service messages are deprecated
 message GetSensorsRequest {
-  string name = 1;
+  string name = 1 [deprecated = true];
```

Comment:
This `name` is marked deprecated, but the names on lines 52 and 59 are not deprecated. Is that on purpose?